rpc/glfs-operations.c

```diff
@@ -174,7 +174,7 @@ glusterBlockCreateEntry(struct glfs *glfs, blockCreateCli *blk, char *gbid,
     ret = -1;
     goto out;
   } else {
-    ret = glfs_ftruncate(tgfd, blk->size);
+    ret = glfs_ftruncate(tgfd, blk->size, NULL, NULL);
```

Comment:
Should we add version check for not breaking with older versions
#if CONFIG_GLUSTERFS_VERSION < X.Y.Z
 ret = glfs_ftruncate(tgfd, blk->size);
#else
ret = glfs_ftruncate(tgfd, blk->size, NULL, NULL);
#end

Reply:
Yeah right. My change will break the build with older versions. Will put a check and update the PR.